docs/README_RECON.md

```diff
@@ -0,0 +1,263 @@
+|-------------|----------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **schema**  | reconcile the schema of source and target.<br> - are same if the source is same <br> - its compatible if the source is different | - **schema_comparison**<br>- **schema_difference**                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| **row**     | reconcile the data only at row level(hash value of the source row is matched with the hash value of the target)                  | - **missing_in_src**(sample rows in target that are not available in source + sample rows in target that doesn't match with source)<br> - **missing_in_tgt**(sample rows in target that are not available in target + sample rows in source that doesn't match with target)<br>  **NOTE**: the report won't differentiate the mismatch and missing here.                                                                                                                                              |
+| **data**    | reconcile the data at row and column level- ```primary_keys``` will play the key role here                                       | - **mismatch_data**(the sample data with mismatches captured at each column and row level )<br> - **missing_in_src**(sample rows that are available in target but missing in src)<br> - **missing_in_tgt**(sample rows that are available in source but are missing in target)<br> - **threshold_mismatch**(configured column will be reconciled based on percentile or threshold boundary or date boundary)<br> - **mismatch_columns**(consolidated list of columns that has mismatches in them)<br> |
```

Comment:
```primary_keys```. I think we should call this ```Join_keys``` since in our config we call it JK's

Reply:
done